src/react/hoc/__tests__/queries/loading.test.tsx

```diff
@@ -408,57 +414,56 @@ describe("[queries] loading", () => {
     })(
       class extends React.Component<ChildProps<{}, Data>> {
         render() {
-          ++count;
-          if (count === 1) {
-            expect(this.props.data!.loading).toBe(true);
-            expect(this.props.data!.allPeople).toBeUndefined();
-          } else if (count === 2) {
-            expect(this.props.data!.loading).toBe(false);
-            expect(this.props.data!.allPeople!.people[0].name).toMatch(
-              /Darth Skywalker - /
-            );
-            // Has data
-            setTimeout(() => render(App));
-          } else if (count === 3) {
-            // Loading after remount
-            expect(this.props.data!.loading).toBe(true);
-            expect(this.props.data!.allPeople).toBeUndefined();
-          } else if (count >= 4) {
-            // Fetched data loading after remount
-            expect(this.props.data!.loading).toBe(false);
-            expect(this.props.data!.allPeople!.people[0].name).toMatch(
-              /Darth Skywalker - /
-            );
-          }
+          currentData = this.props.data!;
           return null;
         }
       }
     );
 
-    const App: React.ReactElement<any> = (
-      <ApolloProvider client={client}>
-        <Container />
-      </ApolloProvider>
+    const ProfiledContainer = profile({
+      Component: Container,
+      takeSnapshot: () => currentData,
+    });
+
+    const App = (
+      <ApolloProvider client={client}>{<ProfiledContainer />}</ApolloProvider>
```

Comment:
```suggestion
      <ApolloProvider client={client}>
        <ProfiledContainer />
      </ApolloProvider>
```

No need for the curlys `{ }` 🙂

Reply:
Good point :)